docs/cloud/managing-airbyte-cloud/review-connection-status.md

```diff
@@ -47,6 +47,6 @@ The stream status allows you to monitor each stream's latest status. The stream
 
 Each stream shows the last record loaded to the destination. Toggle the header to display the exact datetime the last record was loaded.
 
-You can [clear](/operator-guides/reset.md) an individual stream without clearing all streams in a connection by clicking the three grey dots next to any stream.
+You can [clear](/operator-guides/clear) an individual stream without clearing all streams in a connection by clicking the three grey dots next to any stream.
```

Comment:
Noticing the missing `.md`

Reply:
yep! the MDs should not be there.  The URLs we render are estension-less
<img width="878" alt="Screenshot 2024-05-30 at 12 05 12 PM" src="https://github.com/airbytehq/airbyte/assets/303226/8145cab6-0c1e-4776-bbd9-61ee6132ae11">